moped-editor/src/views/projects/projectView/ProjectMilestones/DataGridDateFieldEdit.js

```diff
@@ -34,7 +34,7 @@ const DataGridDateFieldEdit = ({ value, hasFocus, id, field, ...props }) => {
 
   return (
     <MobileDatePicker
-      sx={{ padding: theme.spacing(1) }}
+      sx={{ mx: theme.spacing(1) }}
```

Comment:
cool! this [notation](https://v5-0-6.mui.com/system/the-sx-prop/#spacing) is a TIL for me. ✨